packages/react-vapor/src/components/diffViewer/tests/DiffViewer.spec.tsx

```diff
@@ -1,5 +1,4 @@
-import {screen} from '@testing-library/dom';
-import {render} from '@testing-library/react';
+import {render, screen} from '@testing-library/react';

```

Comment:
```suggestion
import {render, screen} from '@testing-utils';

```

Reply:
![image](https://user-images.githubusercontent.com/966550/122935986-cf0b7600-d33e-11eb-8a0e-1150be664cc0.png)

Do I also change it for its files there?

Reply:
Yes that would be great! thanks 😃